.github/workflows/examples-components.yml

```diff
@@ -185,39 +185,27 @@ jobs:
           WASH_REG_USER: ${{ github.repository_owner }}
           WASH_REG_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          wash push \
-            --config=${{ steps.meta.outputs.wasmcloud-toml-path }} \
-            ghcr.io/${{ github.repository_owner }}/component-${{ matrix.project.name }}:${{ github.sha }} \
-            artifacts/${{ steps.meta.outputs.bin-name }}
+          wash push ghcr.io/${{ github.repository_owner }}/component-${{ matrix.project.name }}:${{ github.sha }} artifacts/${{ steps.meta.outputs.bin-name }}
```

Comment:
no more config?

Reply:
yep, removed this -- the problem didn't turn out to be there -- `registry` is coming through as empty somehow, which is what made me think it was the `\`s. 

See [this failed run](https://github.com/wasmCloud/wasmCloud/actions/runs/8632918526/job/23665073246#step:6:14) for the error message. Clearly something is wrong if `wash push` is not reading the two readily provided arguments.